lib/user/user-delegate.js

```diff
@@ -0,0 +1,106 @@
+var q = require('q')
+  , $fh = require('fh-mbaas-api')
+  , _ = require('lodash');
+
+var Delegate = function(guid) {
+  this.guid = guid;
+};
+
+Delegate.prototype.xhr = function(_options) {
```

Comment:
Can you rename this to MbaasServiceProxy and add a description.

I think that makes more sense than Delegate.

You could also rename the file.